<launch>
	<arg name="map" />

	<node name="map_server" pkg="map_server" type="map_server" args="$(find odrm_graph)/maps/$(arg map).yaml"/>

	<node name="tf_world_map" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 world map 100" />
	<node name="tf_map_base_link" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 map base_link 100" />
	<node name="costmap_2d_node" pkg="costmap_2d" type="costmap_2d_node" />
	<rosparam file="$(find odrm_graph)/config/costmap_common_params.yaml" command="load" ns="costmap_2d_node/costmap" />
    <rosparam file="$(find odrm_graph)/config/global_costmap_params.yaml" command="load" ns="costmap_2d_node/costmap" />
	<node name="pub_footprint" pkg="odrm_graph" type="pub_footprint.py" output="screen">
			<param name="radius" value="2.0" />
	</node>

	<node name="roadmap_server" pkg="odrm_graph" type="roadmap_server.py">
		<param name="graph_pkl" value="$(find odrm_eval)/res/$(arg map)_100_4096.pkl" />
		<param name="cache_dir" value="$(find odrm_graph)/cache" />
	</node>
</launch>